.eslintrc

```diff
@@ -1,13 +1,17 @@
 {
   "parser": "babel-eslint",
-  "extends": ["@jokeyrhyme/es2015"],
+  "extends": ["eslint-config-airbnb"],
   "ecmaFeatures": {
   },
 
   "env": {
+    "browser": true,
+    "node": true
   },
 
-  "globals": {},
+  "globals": {
+    "document" : false
```

Comment:
This was one of three options from this answer
https://stackoverflow.com/questions/41858052/solving-linter-error-no-undef-for-document